CHANGELOG/CHANGELOG-5.3.md

```diff
@@ -0,0 +1,309 @@
+# Release notes for 5.3.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 5.2.0
+
+## Changes by Kind
+
+### Feature
+
+- Add a new `--automaxprocs` flag to set the `GOMAXPROCS` environment variable to match the configured Linux container CPU quota. ([#1379](https://github.com/kubernetes-csi/external-provisioner/pull/1379), [@nixpanic](https://github.com/nixpanic))
+- Promoted the feature gate `HonorPVReclaimPolicy` to GA. ([#1328](https://github.com/kubernetes-csi/external-provisioner/pull/1328), [@carlory](https://github.com/carlory))
+- Users can now provide a secret name and namespace during provision by passing the correct storage class parameters: "provisioner-secret-name" and "provisioner-secret-namespace" ([#274](https://github.com/kubernetes-csi/external-provisioner/pull/274), [@ggriffiths](https://github.com/ggriffiths))
```

Comment:
Something wrong with the release note generator? This is a really old PR, merged 6 years ago.